app/konnect/configure/servicehub/manage-services.md

```diff
@@ -1,152 +1,218 @@
+    can share any Service with any role. For more information, see [
+    Manage Teams, Roles, and Users](/konnect/org-management/teams-roles-users/#entity-and-role-sharing).
+
+1. Click **Share Service** to save.
+
+### Delete a Service
 
-2. Select a Service from the list.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-3. In the top right of the overview page, click the **Actions** menu and select
+1. In the top right of the overview page, click the **Actions** menu and select
 **Delete Service**.
 
-4. In the dialog that appears, confirm that you want to delete this service.
+1. In the dialog that appears, confirm that you want to delete this service.
 
-## Service Versions and Implementations
+## Service versions
 
-### Create a New Service Version
+Every Konnect Service version is associated with one runtime group.
+Any configurations for the Service version, such as implementations, plugins,
+and routes, will also be associated with the same runtime group.
 
-1. From the left navigation menu, click **Services**.
+If a Service has multiple Service versions, each Service version can be
+associated with a different runtime group, or with the same runtime group.
+Through its versions, a Service can made be available in multiple environments,
+simply by assigning Service versions to a different runtime groups.
 
-2. Select a Service from the dropdown menu.
+A common use case for this is environment specialization. For example, if you
+have three runtime groups for `development`, `staging`, and `production`, you
+can manage which environment the Service is available in by assigning that group
+at creation time. So, you might create a v0 in `development`, then a v0.1 in
+`staging`, and a finally a v1 in `production`.
 
-3. Navigate to **Versions**, and click **+ New Version**.
+{:.note}
+> **Note:** You can't move a Service version from one runtime group to another.
+Instead, create a new version of the Service in the new environment when you're
+ready to move to it.
 
-4. Enter a version name and click **Create** to save.
+### Create a new Service version
 
-### Delete a Service Version
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-1. From the left navigation menu, click **Services**.
+1. Navigate to **Versions**, and click **+ New Version**.
 
-2. Select a Service from the dropdown menu.
+1. Enter a **Version Name**.
 
-3. Navigate to **Versions**.
+    A version name can be any string containing letters, numbers, or characters;
+    for example, `1.0.0`, `v.1`, or `version#1`. A Service can have any number of
+    versions.
+
+1. Select a runtime group.
+
+    Choose a group to limit this version to a specific group of runtime
```

Comment:
I don't think the word "limit" is the right word here. It sounds like we are imposing a limitation on this version which isn't true.
Maybe:
```suggestion
    Choose a group to deploy this version to a specific group of runtime
```